src/app/components/trade/SwapPanel.tsx

```diff
@@ -144,67 +144,45 @@ export default function SwapPanel() {
       executionTrade.outputAmount.currency.symbol || 'OUT',
     );
 
-    if (executionTrade.isQueue) {
-      //handle queue
-      if (executionTrade.inputAmount.currency.symbol === 'sICX') {
-        bnJs
-          .inject({ account })
-          .sICX.swapToICX(BalancedJs.utils.toLoop(new BigNumber(executionTrade.inputAmount.toExact())))
-          .then((res: any) => {
-            setShowSwapConfirm(false);
-            addTransaction(
-              { hash: res.result },
-              {
-                pending: message.pendingMessage,
-                summary: message.successMessage,
-              },
-            );
-            handleTypeInput('');
-            handleTypeOutput('');
-          })
-          .catch(e => {
-            console.error('error', e);
-          })
-          .finally(() => {
-            window.removeEventListener('beforeunload', showMessageOnBeforeUnload);
-            changeShouldLedgerSign(false);
-          });
-      } else {
-        bnJs
-          .inject({ account })
-          .Staking.stakeICX(account, BalancedJs.utils.toLoop(new BigNumber(executionTrade.inputAmount.toExact())))
-          .then((res: any) => {
-            setShowSwapConfirm(false);
-            addTransaction(
-              { hash: res.result },
-              {
-                pending: message.pendingMessage,
-                summary: message.successMessage,
-              },
-            );
-            handleTypeInput('');
-            handleTypeOutput('');
-          })
-          .catch(e => {
-            console.error('error', e);
-          })
-          .finally(() => {
-            window.removeEventListener('beforeunload', showMessageOnBeforeUnload);
-            changeShouldLedgerSign(false);
-          });
-      }
+    if (executionTrade.inputAmount.currency.symbol === 'ICX') {
+      bnJs
+        .inject({ account })
+        .Router.swapICX(
+          BalancedJs.utils.toLoop(new BigNumber(executionTrade.inputAmount.toExact())),
```

Comment:
`toLoop` can accept the string value. so please use `executionTrade.inputAmount.toFixed()`.